Directory.Build.props

```diff
@@ -8,7 +8,7 @@
     <!--
     <VersionSuffix>pre</VersionSuffix>
     -->
-    <TargetFrameworks>netstandard2.1;net6.0;net7.0;net8.0;</TargetFrameworks>
+    <TargetFrameworks>netstandard2.1;net8.0;</TargetFrameworks>
```

Comment:
Should net 9 reside here?

Reply:
Yep, .NET 9 will be added after the official release.